packages/sycamore-macro/tests/component/inline-props-pass.rs

```diff
@@ -75,4 +75,9 @@ fn PropsWithVariousImplGenerics(
     }
 }
 
+#[component(inline_props, derive(Clone))]
```

Comment:
Could you add a dummy prop field and also test that the prop actually implements the Clone trait? You can create an instance of the prop type with the automatically generated builder API.